core/common/src/main/java/alluxio/AlluxioEvent.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Defines and Manages well-known Alluxio events.
+ *
+ * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+ * !!! - Keep increasing the event-id for new events. !!!
+ * !!! - Don't reassign event ids.                    !!!
+ * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+ */
+public enum AlluxioEvent {
+  // Alluxio master events.
+  MasterProcessStarting(1000, EventType.INFO, "Master process starting."),
+  JournalSystemStarted(1001, EventType.INFO, "Journal system started. %s"),
+  MasterIsPrimary(1002, EventType.INFO, "Master process is now the primary."),
```

Comment:
We may need to be more precise about these events. For example, I'd imagine the sequence of events could be the following. I don't know how these events map to this sequence:
process started -> journal started -> primacy gained -> serving started -> ... -> primacy lost -> serving stopped -> process stopped

Also, something like "MasterIsPrimary" seems more like a state instead of an event. This could be "primacy gained" if that is what this means.

We may also want to be precise of past tense and present tense. For example, "process starting" is different from "process started". I feel like past tense might be more clear what it means. Like, "journal catchup started" -> "journal catchup finished".

Reply:
Incorporated these to new event names. PTAL.